shell/platform/darwin/ios/BUILD.gn

```diff
@@ -185,7 +175,6 @@ source_set("flutter_framework_source") {
     "AudioToolbox.framework",
```

Comment:
Completely unrelated, but why in the world do we link in AudioToolbox? I think this is code I wrote my first week at Google :/

Reply:
Let's find out https://github.com/flutter/engine/pull/34916

Reply:
The platform plugin interface does provide an API for playing a sound:
https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm#L96